src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelectorAll('table tr');
+
+const cloneTr = [...table];
+
+cloneTr.forEach((row) => {
+  const dublicateTh = row.children[1].cloneNode(true);
```

Comment:
```suggestion
  const duplicateTh = row.children[1].cloneNode(true);
```
typo
I'd recommend you to use Code Spell Checker VSCode extension
FYI